data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -297,6 +303,19 @@ private static OpenApiOptions getOpenApiOptions(
     return options;
   }
 
+  private static Info createApplicationInfo() {
+    return new Info()
+        .title(StringUtils.capitalize(VersionProvider.CLIENT_IDENTITY))
+        .version(VersionProvider.IMPLEMENTATION_VERSION)
+        .description(
+            "A minimal API specification for the beacon node, which enables a validator "
+                + "to connect and perform its obligations on the Ethereum 2.0 phase 0 beacon chain.")
```

Comment:
interesting that this says phase0, maybe we should update that...

Reply:
Changed to just `obligations on the Ethereum beacon chain`.